### PR TITLE
Add Laravel Scout guide

### PR DESCRIPTION
**Pull request by Strift**
# Pull Request

Solves #2713 

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: learn/cookbooks/laravel-scout.mdx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+<Capsule intent="note">
+This comes from Docker [Bridge network driver](https://docs.docker.com/network/drivers/bridge/#differences-between-user-defined-bridges-and-the-default-bridge) behavior.
+</Capsule>
+
+### Running in production
+
+For production use cases, we recommend using a managed Meilisearch via [Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=laravel&utm_source=docs&utm_medium=laravel-scout-guide). On Meilisearch Cloud, you can find your host URL in your project settings.
+
+<Capsule intent="tip">
+Read our [Meilisearch Cloud quick start](/learn/getting_started/cloud_quick_start?utm_campaign=laravel&utm_source=docs&utm_medium=laravel-scout-guide).
```
Comment (guimachiavelli) [inline learn/cookbooks/laravel-scout.mdx:71]:
```suggestion
Read the [Meilisearch Cloud quick start](/learn/getting_started/cloud_quick_start?utm_campaign=laravel&utm_source=docs&utm_medium=laravel-scout-guide).
```

File: learn/cookbooks/laravel-scout.mdx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+</Capsule>
+
+## Making Eloquent models searchable
+
+With Scout installed and configured, add the `Laravel\Scout\Searchable` trait to your Eloquent models to make them searchable. 🎉
+
+This trait will use Laravel’s model observers to keep the data in your model in sync with Meilisearch.
```
Comment (guimachiavelli) [inline learn/cookbooks/laravel-scout.mdx:78]: Call me heartless. I removed your emoji.

More seriously, I think these two work better as a single paragraph so it reinforces the connection between "this trait" and the trait you're referring to.

```suggestion
With Scout installed and configured, add the `Laravel\Scout\Searchable` trait to your Eloquent models to make them searchable. This trait will use Laravel’s model observers to keep the data in your model in sync with Meilisearch.
```

File: learn/cookbooks/laravel-scout.mdx [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+To synchronize your index settings, run the following command:
+
+```sh
+php artisan scout:sync-index-settings
+```
+
+## Example usage
+
+We built an example application to demonstrate how to use Meilisearch with Laravel Scout. It showcases an app-wide search in a CRM (Customer Relationship Management) application.
```
Comment (guimachiavelli) [inline learn/cookbooks/laravel-scout.mdx:178]:
```suggestion
You built an example application to demonstrate how to use Meilisearch with Laravel Scout. It showcases an app-wide search in a CRM (Customer Relationship Management) application.
```

File: learn/cookbooks/laravel-scout.mdx [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+```sh
+php artisan vendor:publish --provider="Laravel\Scout\ScoutServiceProvider"
+```
+
+This command should create a new configuration file in your application directory: `config/scout.php`.
+
+## Configuring the Laravel Scout driver
+
+Let’s configure Laravel Scout to use the Meilisearch driver. First, install the dependencies required to use Scout with Meilisearch via Composer:
```
Comment (guimachiavelli) [inline learn/cookbooks/laravel-scout.mdx:43]:
```suggestion
Now you need to configure Laravel Scout to use the Meilisearch driver. First, install the dependencies required to use Scout with Meilisearch via Composer:
```

File: learn/cookbooks/laravel-scout.mdx [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+Laravel comes with out-of-the-box full-text search capabilities via Laravel Scout.
+
+To enable it, navigate to your Laravel application directory and install Scout via the Composer package manager:
+
+```sh
+composer require laravel/scout
+```
+
+After installing Scout, you need to publish the Scout configuration file. You can do this by running the following artisan command:
```
Comment (guimachiavelli) [inline learn/cookbooks/laravel-scout.mdx:33]:
```suggestion
After installing Scout, you need to publish the Scout configuration file. You can do this by running the following `artisan` command:
```

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
A few minor changes, but looks good. Very concise and to the point. If you have the time, I think the only part that might need some extra work is the subsections of "Configuring the laravel scout driver".

**Comment by guimachiavelli** [comment]
Oh, and do run linters to check for style errors and broken links. I see the action already reported a few broken links we need to fix before merging.

File: learn/cookbooks/laravel-scout.mdx [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+```sh
+SCOUT_DRIVER=meilisearch
+# Use the host below if you're running Meilisearch via Laravel Sail
+MEILISEARCH_HOST=http://meilisearch:7700
+MEILISEARCH_KEY=masterKey
+```
+
+### Local development
+
+Laravel’s official Docker development environment, Laravel Sail, comes with a Meilisearch service out-of-the-box. Please note that when running Meilisearch via Sail, Meilisearch’s host is `http://meilisearch:7700`.
```
[the remark this answers is not in the document]
Comment (Strift) [reply learn/cookbooks/laravel-scout.mdx]: A "service" is what Docker compose calls each of the "instances" that are run. I understand the fear for unnecessary confusion. 

But on the other hand, for someone familiar with Docker compose, it's  clear. It's the name that they should use to google or search in the Docker docs

**Review by Strift — COMMENTED** [review]
> looks good. Very concise and to the point
😌 😌 😌 

Thanks for the review! I accepted all suggestions to make the tone more in line with what we have already.

Also rewrote the sections you suggested. I leave it up to you to close the relevant discussion if you think we're good 🙏

**Review by guimachiavelli — DISMISSED** [review]
LGTM, but there are a couple of things we need to do before we can merge this:

1. Fix the broken link errors: I believe this means you'll have to remove the fathom stuff until we can get basement to improve the checker, otherwise github won't let us merge into `main`
2. Fix the merge conflict: we recently merged the other laravel guide which altered the same lines of `sidebar_learn.json`. Should be a simple matter of ensuring both objects (the menu items for each guide) are present and well-formatted

**Comment by Strift** [comment]
@guimachiavelli Fixed both 👍

**Review by CaroFG — APPROVED** [review]
LGTM!

**Pull request merged**